tests-legacy/Integration/PrestaShopBundle/Controller/Api/Improve/Design/PositionsControllerTest.php

```diff
@@ -49,6 +50,8 @@ protected function setUp()
         Module::clearStaticCache();
 
         parent::setUp();
+        $this->logIn();
+        unset($_SERVER['REMOTE_ADDR']);
```

Comment:
No explanation? :thinking:

Reply:
I'm still trying to make legacy tests work, there are many problems with them with the new login system :confused: this unset apparently fixes some of them, because of the following condition in legacy `Tools`: https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Tools.php#L3505

Reply:
so better to use 
```
  <server name="REMOTE_ADDR" value=""/>
```

Reply:
I'll try it, thanks!

Reply:
@PierreRambaud maybe you also have an idea why the tests are still failing? I tried so many solutions, many of them are working already, but some still don't wan't to run... :confused: